lib/performance_data/statistics.rb

```diff
@@ -0,0 +1,247 @@
+            searches: searches
+          }
+        end
+      end
+
+      @search_terms ||= stats
+    end
+
+    def lead_metrics
+      pageview_data = performance_data_for(page_views, part_urls)
+      search_data = performance_data_for(searches, part_urls)
+      problem_data = performance_data_for(problem_reports, part_urls)
+      search_term_data = performance_data_for(search_terms, [])
+      if is_multipart
+        return PerformanceData::MultiPartMetrics.new(
+          unique_pageviews: pageview_data,
+          exits_via_search: search_data,
+          problem_reports: problem_data,
+          search_terms: search_term_data
+        )
+      else
+        return PerformanceData::Metrics.new(
+          unique_pageviews: pageview_data,
+          exits_via_search: search_data,
+          problem_reports: problem_data,
+          search_terms: search_term_data
+        )
+      end
+    end
+
+    def per_page_metrics
+      per_page_metrics = {}
+
+      part_urls.each do |path|
+        per_page_metrics[path] = metrics_for(path)
+      end
+
+      per_page_metrics
+    end
+
+  private
+
+    def get_part_urls(content, slug)
+      details = content.fetch("details")
+      return [] unless details.key?("parts")
+
+      parts = details.fetch("parts")
+
+      part_urls = parts.map! do |part|
+        if part.key?("web_url")
+          URI(part["web_url"]).path
+        else
+          "/#{slug}/#{part['slug']}"
+        end
+      end
+      part_urls.unshift(slug.insert(0, "/"))
+    end
+
+    # For individual page metrics, we don't include search terms
+    # because they cannot be broken down into multiple url parts.
+    # In fact the search terms response does not return a `pagePath`,
+    # but a `searchKeyword` (e.g. `employer access`)
+    #
+    # Example response for searches:
+    #
+    # {
+    #     "data": [
+    #         {
+    #             "pagePath": "/tax-disc",
+    #             "values": [
+    #                 {
+    #                     "_count": 4,
+    #                     "_end_at": "2014-09-03T00:00:00+00:00",
+    #                     "_start_at": "2014-09-02T00:00:00+00:00",
+    #                     "searchUniques:sum": 71
+    #                 }
+    #             ]
+    #         }
+    #     ]
+    # }
+    #
+    # Example response for search_terms:
+    #
+    # {
+    #     "data": [
+    #         {
+    #             "_count": 8,
+    #             "_group_count": 8,
+    #             "searchKeyword": "employer access",
+    #             "searchUniques:sum": 126,
+    #             "values": [{
+    #                            "_count": 1,
+    #                            "searchUniques:sum": 126,
+    #                            "_end_at": "2014-09-03T00:00:00+00:00",
+    #                            "_start_at": "2014-09-02T00:00:00+00:00"
+    #                        }]
+    #         }
+    #     ]
+    # }
+
+    def metrics_for(path)
```

Comment:
maybe `per_page_metrics_for` or `per_part_metrics_for` ?  To make it super clear this isn't what we want for the main metrics?